test/src/main/java/org/apache/accumulo/test/functional/ZooMutatorIT.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.test.functional;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.accumulo.core.client.Accumulo;
+import org.apache.accumulo.core.clientImpl.ClientContext;
+import org.apache.accumulo.core.conf.Property;
+import org.apache.accumulo.fate.zookeeper.ZooReaderWriter;
+import org.apache.accumulo.harness.AccumuloClusterHarness;
+import org.apache.commons.codec.digest.DigestUtils;
```

Comment:
commons-codec is not currently a direct dependency of the test module. I don't think it's worth adding it here, since the use of sha1 will likely trigger the secbugs plugin anyway. I would use Guava's Hasher.sha512 instead. It has a convenient hashString method.

Reply:
I didn't realize it was not a direct dep, good catch.  I started off using Guava Hasher, but I think it was marked `@Beta` so I looked for an alternative.  I will look for some other way to do this.

Reply:
Tried another way in 00b3542

Reply:
It's fine to use the Beta method for a test. We don't need to worry about version incompatibilities in our test cases. It would be shorter code. Also, we already use it in other tests, and even some non-tests (which we shouldn't, but we do).

Reply:
I was thinking if anyone wants to try to remove more `@Beta` usage in the future, this is one less place to deal with it. I like not introducing new `@Beta` usage intentionally even if there is existing usage.

Reply:
> but I still think it's fine for Test code

I suppose that its ok, if we would never want to remove @Beta usage from test code.  In that case its not creating possible future work.  I will switch it, Guava's Hashing class is very pleasant to use too bad its beta.